src/org/labkey/targetedms/pipeline/TargetedMSImportTask.java

```diff
@@ -85,7 +77,7 @@ public RecordedActionSet run() throws PipelineJobException
         return new RecordedActionSet();
     }
 
-    public static class Factory extends AbstractTaskFactory<AbstractTaskFactorySettings, ExperimentExportTask.Factory>
```

Comment:
I don't know how the FactoryType got set to ExperimentExportTask.Factory!  ExperimentExportTask is a task in the PanoramaPublic pipeline.